### PR TITLE
Upgrade to dotnet 7 & project refactoring

### PR DESCRIPTION
**Pull request by Jossec101**
- feat: bump to dotnet 7 environment (not libs) refactor: renamed FundsManager namespaces and solutions to NodeGuard refactor: removed unused docker-compose files
- feat: upgrade to dotnet 7 libraries and all libraries to latest version refactor: refactored opentelemetry setup feat: bump dockerfile to dotnet 7
- refactor(InternalWallet.cs): update comment to reflect change from FundsManager to NodeGuard for clarity refactor(NodeGuard.csproj): update UserSecretsId and RunConfiguration to reflect project name change from FundsManager to NodeGuard refactor(_Layout.cshtml): remove unused blazored-toast.min.css import to improve load times
- refactor: removed unused files

### CODE REVIEW

File: src/Program.cs [hunk 1]
```diff
@@ -248,38 +249,36 @@ public static void Main(string[] args)
 
             if (Constants.OTEL_EXPORTER_ENDPOINT != null)
             {
-                builder.Services
-                    .AddOpenTelemetryTracing((builder) => builder
+                builder.Services.AddOpenTelemetry().WithMetrics(builder => builder
                         // Configure the resource attribute `service.name` to MyServiceName
                         //.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("BtcPayServer"))
```
Comment (RodriFS) [inline src/Program.cs:254]: Can we delete this?

File: src/Program.cs [hunk 2]
```diff
@@ -248,38 +249,36 @@ public static void Main(string[] args)
...
-                        .AddEntityFrameworkCoreInstrumentation()
-                        .AddQuartzInstrumentation()
-                    );
-
-                builder.Services
-                    .AddOpenTelemetryMetrics(builder => builder
+                    )
+                    .WithTracing((builder) => builder
                         // Configure the resource attribute `service.name` to MyServiceName
                         //.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("BtcPayServer"))
```
Comment (RodriFS) [inline src/Program.cs:268]: and this?

**Review by RodriFS — APPROVED** [review]

File: src/Program.cs (hunk 1 above)
Comment (Jossec101) [inline src/Program.cs:254]:
```suggestion

```

File: src/Program.cs (hunk 2 above)
Comment (Jossec101) [inline src/Program.cs:268]:
```suggestion

```

**Pull request merged**